assets/stylesheets/components/_inline-list.scss

```diff
@@ -0,0 +1,23 @@
+.inline-list {
+  list-style: none;
+  padding-left: 0;
+}
+
+.inline-list--item {
+  // at media "tablet or medium"
+  display: inline;
+}
+
+// at media "tablet or medium" for all of this bit
+.inline-list__slash {
+
+  .inline-list--item {
+    &:after {
+      content: "/";
+    }
+
+    &:last-child:after {
```

Comment:
Don't think IE8 support `:last-child`. It supports `:first-child` as it's part of CSS 2.1 spec but `:last-child` is CSS3.

We could say we're happy for IE8 just to show a final trailing slash though?